consult.el

```diff
@@ -4587,6 +4587,23 @@ If NORECORD is non-nil, do not record the buffer switch in the buffer list."
                                        :as #'consult--buffer-pair)))
   "Buffer candidate source for `consult-buffer'.")
 
+(defvar consult--source-tab-buffer
+  `(:name     "Tab buffer"
+    :narrow   ?b
+    :category buffer
+    :face     consult-buffer
+    :history  buffer-name-history
+    :state    ,#'consult--buffer-state
+    :default  t
```

Comment:
- :default should not be set here. :default is only used is used by consult--source-buffer.